generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -157,37 +157,37 @@ message VirtualInterface {
   repeated GatewayAddress gateway_addresses = 8;
 }
 
-message IPv6Addr {
+message In6Addr {
```

Comment:
Same as removing extra tokenization for ALLCAPS. The C name for these is `in6_addr`, so I'm switching back to that.